nuxeo-retention-core/src/main/java/org/nuxeo/ecm/retention/service/RetentionComponent.java

```diff
@@ -312,6 +314,7 @@ public void endRetention(Record record, RetentionRule rule, CoreSession session)
         executeRuleAction(rule.getEndAction(), record.getDoc(), session);
         notifyEvent(session, RETENTION_EXPIRED_EVENT, record.getDoc());
         record.setStatus(RETENTION_STATE.expired.name());
+        session.setRetentionActive(record.getDoc().getRef(), false);
```

Comment:
Since Nuxeo 9.10 there's a retention active flag at system level and I thought it was a good idea to toggle this flag here.
But I reverted this changes in my last commit because I need to talk with the core team before.